app/helpers/application_helper.rb

```diff
@@ -88,7 +88,7 @@ def header_nav_separator
   end
 
   def icon(name, size: '1x')
-    tag.i '', class: "fas fa-#{name} fa-#{size}"
+    tag.i '', class: "#{name} fa-#{size}"
```

Comment:
Suggest we keep it how it was before but only make the `fas` bit configurable, by making the method definition be:

```ruby
def icon(name, variant: 'fas', size: '1x')
```

 … then this line can be:

```ruby
    tag.i '', class: "#{variant} fa-#{name} fa-#{size}"
```

This makes for a nicer _default_ experience, and also doesn't tie the icon helper to Font Awesome specifically. (In theory, the icon `name` passed in should be generic enough that substituting with a different icon set later is an easier job).

Reply:
Done.